Packs/CTIX/ReleaseNotes/2_2_10.md

```diff
@@ -0,0 +1,10 @@
+#### Integrations
+
+##### Cyware Threat Intelligence eXchange
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+
+##### CTIX v3
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+- Added a ctix-get-vulnerability-data command.
+- Added a cve command.
+- Update documentation for fields that are actually required.
```

Comment:
Perhaps "Added a default value for required argument fields" ?

Reply:
Thanks, I made that change.